src/engine/options.rs

```diff
@@ -0,0 +1,84 @@
+// This file is part of rust-web/twig
+//
+// For the copyright and license information, please view the LICENSE
+// file that was distributed with this source code.
+
+//! Twig configuration options.
+
+use std::path::{Path, PathBuf};
+use extension::escaper;
+use extension::optimizer;
+
+pub type Autoescape = escaper::Mode;
+pub type Optimizations = optimizer::Mode;
+
+#[derive(Debug, PartialEq, Clone, Copy)]
+pub enum Charset {
+    UTF8
```

Comment:
Hm, this is one of those cases where I would say "screw them!" and not support anything else ;)